### ISSUE TITLE
copy story for the variant cover sequence

### ISSUE THREAD

**Issue by jochengcd** [GrandComicsDatabase/gcd-django#570]
When adding a variant issue, there is no option to copy from another issue.  Variants are limited so that only a cover sequence can be added to it, but often I find that I am adding the same cover sequence with the same artist, feature, genre, and characters as existing issues.

**Comment by J-Garcke-SCAI** [comment]
The workflow needs currently that the issue has an issue_id, so first the issue needs to approved before we can copy stories.

**Comment by jochengcd** [comment]
The workflow needs currently that the issue has an issue_id, so first the issue needs to approved before we can copy stories.
